src/utils/SortScenarioListUtils.js

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+// SORT
+export const testSort = (scenarioList) => {
+  const sortedList = sortScenarioList(scenarioList.data.slice());
+  console.log(sortedList);
+};
+
+function indexOfById (list, id) {
+  for (let i = 0; i < list.length; ++i) {
+    const item = list[i];
+    if (item.id === id) {
+      return i;
+    }
+  }
+  return -1;
+}
```

Comment:
This function can be replaced by the **find** method of arrays:
`list.find(item => item.id === id)`
If no items match the expected condition, then _undefined_ is returned.
More info here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
Nice, I don't know all the these algo yet. I'll update it.

Reply:
Also, it's more `findIndex()` than `find()`

Reply:
don't forget to remove the function if it's not used ;)